stable/clamav/values.yaml

```diff
@@ -12,6 +12,9 @@ image:
 nameOverride: ""
 fullnameOverride: ""
 
+daemonset:
+  enabled: true
```

Comment:
by default, this should be disabled. Its a special use-case that needs a deamonset.